### ISSUE TITLE
`Error: Class "DocumentCreateManagerController" not found` when enable_template_picker_in_tree is off

### ISSUE THREAD

**Issue by Mark-H** [modxcms/revolution#16004]
### Summary

When the template picker in 3.0.0-rc1 is DISABLED, or when using Collections or a quick dashboard link to create a new document, an error is thrown. 

### Step to reproduce

Try to navigate to the resource/create controller with a pre-defined class key, e.g.:

- `https://localhost/3.x/manager/?a=resource/create&class_key=MODX\Revolution\modDocument&parent=1&context_key=web`

URLs like this are generated with the template picker off and right clicking in the tree > create document here, or through Collections or Quickstart Buttons. 

### Observed behavior

![afbeelding](https://user-images.githubusercontent.com/312944/151014321-ac2a99f7-e849-4e93-9f64-7ed76030653e.png)

```
Error: Class "DocumentCreateManagerController" not found

Errors:

    ResourceManagerController::getInstance()
    MODX\Revolution\modManagerResponse->outputContent()
    MODX\Revolution\modManagerRequest->prepareResponse()
    MODX\Revolution\modManagerRequest->handleRequest()
```

What's interesting though: **it does work for weblinks, static resources and symlinks**. So it seems to be document-specific. 

When removing the class_key URL parameter, it also works fine. 

### Expected behavior

Allow creating a new document without the template picker.

### Related issue(s)/PR(s)
N/a

### Environment
3.0.0-rc1

**Comment by JoshuaLuckers** [comment]
@Mark-H this has been resolved in PR #15997

**Comment by Mark-H** [comment]
Ah, totally missed that one. Great!
